A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/EICAS/ECAM/A320_Neo_UpperECAM.js

```diff
@@ -948,6 +948,19 @@ var A320_Neo_UpperECAM;
                             }
                         ]
                     },
+                    {
+                        name: "F/CTL",
+                        messages: [
+                            {
+                                id: "to_flaps_disagree",
+                                message: "FLAP/MCDU DISAGREE",
+                                level: 2,
+                                flightPhasesInhib: [5, 6, 7, 8], // TODO
```

Comment:
Can you clarify this TODO message ?

Reply:
It's a copy over from. All lines handling TO CONFIG etc have it and since it's the same code, I kept it since someone made plans. It's regarding the flight phases, maybe @beheh knows more about that.

Reply:
It should be this:

```suggestion
                                flightPhasesInhib: [1, 4, 5, 6, 7, 8, 9, 10],
```